apps/docs/docs/ref/05-command-line-reference.mdx

```diff
@@ -0,0 +1,46 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Verse Command Line Reference
+
+The Verse cli helps with design-time development and deployment tasks. It's primarily used to manage Migrations.
+
+## Installing the Verse CLI
+
+You should install the Verse cli locally as a dev dependency. The Verse cli can be installed by:
+
+```bash npm2yarn
+npm install @operativa/verse-cli --save-dev
+```
+
+Verify your installation by running:
+
+```bash npm2yarn
+npx verse
```

Comment:
This should be `npm`

Reply:
If we specify `npm` then we must add a `verse` script to `package.json`. Using `npx` allows it to just be run from command line without additional changes.